src/widgets/image/lv_image.c

```diff
@@ -128,7 +128,9 @@ lv_obj_t * lv_image_create(lv_obj_t * parent)
 {
     LV_LOG_INFO("begin");
     lv_obj_t * obj = lv_obj_class_create_obj(MY_CLASS, parent);
-    lv_obj_class_init_obj(obj);
```

Comment:
How about check NULL obj in `lv_obj_class_init_obj` and return directly?

Reply:
check in `lv_obj_class_init_obj` is better, I have updated the patch